lib/instruments.js

```diff
@@ -173,17 +175,18 @@ class Instruments {
 
     // start waiting for instruments to launch successfully
     this.addSocketConnectTimer(this.launchTimeout.global, 'global', async () => {
-      await killAllInstruments();
+      this.proc && this.proc.kill();
```

Comment:
Same here. Since it's in two places, it's probably worth making a method that does it.